functions/tips-gcp-apis/main.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START functions_tips_gcp_apis]
+import os
+
+from google.cloud import pubsub_v1
+
+
+# Create a global Pub/Sub client to avoid unneeded network activity
+pubsub = pubsub_v1.PublisherClient()
+
+
+def gcp_api_call(request):
```

Comment:
No test for this sample?

Reply:
Will do in a follow-up PR. (See b/261602561 internally.)